BookOfHabits/BookOfHabitsMicroservice.Domain.Entity/Enums/OptionCoins.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BookOfHabitsMicroservice.Domain.Entity.Enums
+{
+    [Flags]
+    public enum OptionCoins
+    {
+        None = 0,
+        FallingLevel = 1,
+        AutoPayment = 2,
+        CheckingLimit = 4,
+        CheckingReport = 8
```

Comment:
Why do you need specific numbers for enums? Why don't you want to use the default?